GAE/src/org/waterforpeople/mapping/serialization/response/MediaResponse.java

```diff
@@ -28,6 +28,7 @@
     private static final Logger log = Logger.getLogger(MediaResponse.class.getName());
     private static FlowJsonObjectReader jsonObjectReader = new FlowJsonObjectReader();
     private static FlowJsonObjectWriter jsonObjectWriter = new FlowJsonObjectWriter();
+    private static FlowJsonObjectWriter jsonObjectWriterNoNulls = new FlowJsonObjectWriter().withExcludeNullValues();
```

Comment:
Does it mean we write null in a json file? I had issues in Android as it skipped null values so wanted to make sure, the method name does not seem clear.

Reply:
The noNull writer writes nothing in the json for null java values.